docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -1,2 +1,101 @@
++
+`IT-0000000001`
++
+`IT-0000000002`
++
+`IT-0000000003`
+
++
+. Click *Ok* to open the case designer.
++
+The designer creates a `WorkDefinition` file that includes the work items required for case definition. These appear as usable objects in the case designer *Object Library*, accessible by clicking the image:3897.png[left arrows] in the upper-left corner of the designer.
++ 
+** Verify that the case designer includes the `Milestones` and `Cases` objects by either checking the *Object Library* dock on the left side of the case designer, or by checking the `WorkDefinition` file under *Work Item Definitions* in the *Project Explorer* window on the left side of the {CENTRAL} workbench.
+. *Define process level and case level variables.* 
++
+Click the image:3140.png[] arrow on the right side of the designer to open the `Properties (BPMN-Diagram)` panel. Click the down arrow next to `Variable Definitions` to open the `Editor for Variable Definitions`.
++
+.IT Orders variables
+image::case-variables.png[Editor for IT Orders variables]
++
+In the IT Orders sample project, you can see that certain variables are identified as `Case File`. A case file variable is stored in the case file itself, and promotes reuse of the case file data rather than copying it across multiple process instances. For example, the `hwSpec` variable is available to all processes within the case, and is accessible directly from the case file even without an existing process instance. 
++
+For more information about the case file, see <<case-management-case-file-con-{context}>>.
+
+. *Define the roles involved in the case.*
+.. Click image:3898.png[] to open the *Properties* menu on the right side of the designer and open the *Editor for Case Roles*. 
+.. Click *Add Case Role* to add case roles.  
++
+The IT Orders sample case management project includes the following roles:
++
+
+.ITOrders Case Roles
+image::case_roles.png[Case Roles]
+
++
+* `owner` - the employee who is making the hardware order request. There can be only one of these roles.
+* `manager` - the employee's manager; the person who will approve or deny the requested hardware. There is only one manager in the IT Orders project.
+* `supplier` - the available suppliers of IT hardware in the system. There is usually more than one supplier.
++
+For more information about creating case roles, see <<case-management-roles-con-{context}>>.
+
+. *Define what work item will be triggered when a new case starts.*
++
+As case definition is an ad hoc process definition, it does not require an explicit start node. Nodes that have no incoming connections and are marked as `Adhoc Autostart` and are automatically triggered when an instance is started.
++
+By default, the case definition designer includes a `Milestone` object configured as `Adhoc autostart` in the palette. Modify this object or replace it with another work item, such as a user task or script task to be triggered when a new case is started.
```

Comment:
is this really true - milestone has set auto start to true? I mean if you drag and drop new milestone node on the canvas will it have auto start set to true?

Reply:
When you create a new case definition there is already a Milestone that is preconfigured as autostart true. Even dragging a new Milestone node on the canvas appears to have a default setting of Adhoc autostart=true. The same applies to Sub Case. This is in the latest build.